### ISSUE TITLE
OTExtension was not found: add OTExtension subdirectory

### ISSUE THREAD

**Issue by code-X-HQM** [encryptogroup/ABY#206]
when I use `make`,  the `/bin` folder is empty.

When I execute `cmake .. -DABY_BUILD_EXE=On`, the terminal will prompt the following error.

-- Configured operating system: LINUX
-- Compiler flags:  -Wall   -O3 -funroll-loops -fomit-frame-pointer -march=core2 -msse4.2 -mpclmul
-- Linker flags: -L/home/～/code/ABY/extern/ENCRYPTO_utils/extern/relic/src/low/curve2251-sse/ 
-- Configured /home/～/code/ABY/extern/ENCRYPTO_utils/extern/relic/include/relic_conf.h.in
-- Configured GMP: -I/usr/include/x86_64-linux-gnu -L/usr/lib/x86_64-linux-gnu/libgmp.so
-- Configured /home/～/code/ABY/extern/ENCRYPTO_utils/extern/relic/doc/relic.doxygen.in
**OTExtension was not found: add OTExtension subdirectory**
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found suitable version "1.74.0", minimum required is "1.66.0") found components: thread system 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/***/code/ABY/build

When I only use `make otextension` and then use `make`, I can get the right result in `/bin`. 

I don't know what causes it, I'm curious.

**Comment by mialuyao** [comment]
I have got the same problem. Have you solved it?

**Comment by code-X-HQM** [comment]
> I have got the same problem. Have you solved it?

Yes, as described in my question, I solved it. You need to install the error-reporting library separately. For example, I had an error installing OTExtension, so I `make otextension` first and then `make`.
